### DISCUSSION TITLE
What’s the Issue❓ [Issues Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#38203] [Projects and Issues]
New updates, new community heroes, and UNIVERSE happening next week! Happy November - let us harvest this month’s Issues mini check-in. 

🦸‍♂️ **Incredible in Issues:** A round of applause for these community citizens 👏
* @Thinkh: For a great self-solve! They came back to [update](https://github.com/orgs/community/discussions/35871) us and gave some tips! 
* @rickstaa: Thank you for coming back to your [feature request](https://github.com/orgs/community/discussions/36119) after checking the documentation. 

🐙**Issues Info**: All updates, info, releases, etc that are Issues-related!
* Latest Update: Auto-archive, board configuration, and bug fixes. [Learn more](https://github.blog/changelog/2022-10-18-the-new-github-issues-october-18th-update/). 

🧹**Housekeeping**: Community on Discussions updates! 
* Share your feedback: @2percentsilk [posted](https://github.com/orgs/community/discussions/36466) a Discussion on the new auto-archiving for your projects. Give it a try and let us know what you think 👂! 
* GitHub Universe is next week: Don’t miss our biggest global developer event in San Francisco 🌉 and virtually November 9-10. [Get the details](https://githubuniverse.com/) 

We hope to see you next week at GitHub Universe!

**Comment by mdAshik922** [comment]
Of course, we hope to see you in GitHub Universe next week!

thank you

**Comment by Preetiraj3697** [comment]
Thankyou. Wait.....

**Comment by pmcristea** [comment]
I'm trying to copy the results from the Ask Copilot but it doesn't paste or copy.
how can I solve this issue?
